Automattic/wp-calypso
# Plan Comparison Grid: Update breakpoints in Onboarding

Related to https://github.com/Automattic/martech/issues/1530

## Proposed Changes

* Previously, the same breakpoints were being applied to both the plan features grid **and** the plan comparison grid
* These changes add new, plan comparison grid specific breakpoints
* Updates the number of plans that we render at each breakpoint

## GIFS
![2023-05-17 16 02 31](https://github.com/Automattic/wp-calypso/assets/5414230/dfa4ff8c-81f1-470d-a8f5-43f5e2298284)

## Testing Instructions


* Check out this branch and run `yarn start`
* Navigate to calypso.localhost:3000/start
* Choose some free domain and land on the plans page
* Verify that the number of plans shown at each breakpoint is correct
* Verify that the width of each plan column and the gutter space between columns is correct

**ONBOARDING FLOW**
- From 1600px to 1281px, show all plans. 
- From 1280px to 1024px, show 4 plans. 
- From 1023px to 880px, show 3 plans
- From 880px to mobile, we show 2 plans. 

## Pre-merge Checklist
